sdk/Sdk.Generators/FunctionExecutor/FunctionExecutorGenerator.Emitter.cs

```diff
@@ -42,6 +43,10 @@ public async ValueTask ExecuteAsync(FunctionContext context)
                                      {{GetMethodBody(functions)}}
                                  }
                              }
+
```

Comment:
Do we need "/// <inheritdoc/>" in line 40?

Reply:
Not required as this type itself is internal. But will add for consistency.